### PR TITLE
[RDS-428] - Add prettier, refactor all styling

### PR DESCRIPTION
**Pull request by roszpun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +60,9 @@
     "yargs": "^16.2.0"
   },
   "devDependencies": {
+    "prettier": "^2.8.8",
```
Comment (rpiszczatowski) [inline package.json:63]: I recommend upgrading versions of added dependencies, especially prettier as its config changes in v3 in a way that you have to be explicit about the plugins you use.

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (rpiszczatowski) [inline .prettierrc.json:1]: You can use our company-wide config by adding a dev dependency:
```
    "redstone-prettier-config": "^1.0.0",
```
This is basically equivalent to the following config:
```
{
  trailingComma: "es5",
  tabWidth: 2,
  semi: true,
  singleQuote: false
}
```

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
```
Comment (rpiszczatowski) [inline .eslintrc.js:1]: When you decide to add .eslintrc.js you should get rid of the eslint config from package.json

**Comment by roszpun** [comment]
Upgraded to prettier 3, included dependency config, removed package eslint config.

**Review by rpiszczatowski — APPROVED** [review]

**Pull request merged**